trieste/acquisition/rule.py

```diff
@@ -594,6 +612,13 @@ def state_func(
                     new_optimization_step=False,
                 )
                 new_point = self._optimizer(search_space, self._acquisition_function)
+                if summary_writer:
+                    with summary_writer.as_default(step=step_number):
+                        batched_point = tf.expand_dims(new_point, axis=0)
+                        value = self._acquisition_function(batched_point)[0][0]
+                        tf.summary.scalar(
+                            f"AsyncGreedy.acquisition_function.maximum_found.{i}", value
```

Comment:
Following the EGO example, this shall be
```suggestion
                            f"AsyncGreedy.acquisition_function.maximum_found.{i+1}", value
```

Reply:
EGO (where we don't always have batches) numbers them "maximum_found", "maximum_found.1", "maximum_found.2", etc.

AsyncGreedy (where we do always have batches) numbers them "maximum_found.0", "maximum_found.1", "maximum_found.2", etc.

I don't think we should worry too much about this until people start giving using it and giving feedback, as we may end up changing the hierarchy of metrics much more significantly.

Reply:
fair enough